frontend/packages/schema-model/src/lib/SchemaModel.test.ts

```diff
@@ -104,7 +104,7 @@ describe('SchemaModel', () => {
     });
 
     it('Throws an error if the root node is not a field node', () => {
```

Comment:
Should this include '...is not a field `or combination` node'?

Reply:
Good catch!

Reply:
Fixed.